packages/primitives-core/src/styled.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import * as React from 'react'
+import {
+  channel,
+  contextTypes,
+  setTheme,
+  testAlwaysTrue,
+  pickAssign,
+  interleave
+} from './utils'
+import { createCss } from './css'
+
+type State = {
+  theme: Object
+} | null
+
+let defaultPickTest = prop => prop !== 'theme' && prop !== 'innerRef'
+
+export function createStyled(
+  StyleSheet: Object,
+  getShouldForwardProp: (
```

Comment:
Could you make this argument into an object in case we need to add more options in the future?